packages/components/Link/src/Link.tsx

```diff
@@ -39,8 +39,9 @@ export const Link = compose<LinkType>({
     return (final: LinkProps, ...children: React.ReactNode[]) => {
       // the event fires twice due to native's implementation of inline link
       const { inline, ...mergedProps } = mergeProps(link.props, final);
+      const supportsTextInText = Platform.OS !== 'android';
 
-      return inline || mergedProps.selectable ? (
+      return supportsTextInText && (inline || mergedProps.selectable) ? (
```

Comment:
This means that inline links are not supported on Android. Is that the intended change? If so you should update the docs and the comment of the inline prop to reflect this.